### PR TITLE
Clear auth info before logging

### PR DESCRIPTION
**Pull request by IndrekV**
### 📒 Description
Removed the auth info from logs

### 🔎 Review hints
No auth info should be present in the logs.

### CODE REVIEW

File: src/https_client.cc [hunk 1]
```diff
@@ -410,6 +410,9 @@ HTTPSResponse HTTPSClient::makeHttpRequest(
         // Request gzip unless downloading files
         poco_req.set("Accept-Encoding", "gzip");
```
Comment (skel35) [inline src/https_client.cc]: I've also noticed this suspicious line of code #3780 as it doesn't seem to influence anything.
I did some testing without this line and I did some code history investigation. I've discovered only that it used to be before sending the request and was moved here in the commit 1edcb957eb9db03ce6dd3d0c4abc9e73bef61641. I tested most of the basic operations (create/stop TE, create/change project/client/tag) and also tested sending feedback, everything seems to work.

So my vote is to simply remove this line.

File: src/https_client.cc [hunk 2]
```diff
@@ -410,6 +410,9 @@ HTTPSResponse HTTPSClient::makeHttpRequest(
...
 
+        // Remove Auth info
+        poco_req.set("Authorization", "");
```
[the remark this answers is not in the document]
Comment (IndrekV) [reply src/https_client.cc]: I agree, this is a better solution.

File: src/https_client.cc (hunk 1 above)
Comment (IndrekV) [reply src/https_client.cc]: I created a separate issue to look into the gzip compression part #3780

**Comment by IndrekV** [comment]
Ready for another round of review

**Pull request merged**